jupyterhub/templates/hub/deployment.yaml

```
@@ -106,7 +106,7 @@ spec:
             - mountPath: /etc/jupyterhub/jupyterhub_config.py
               subPath: jupyterhub_config.py
               name: config
-            - mountPath: /usr/local/lib/python3.6/dist-packages/z2jh.py
+            - mountPath: /etc/jupyterhub/z2jh/z2jh.py
```

Let's put in directly in /etc/jupyterhub alongside the cull script and jupyterhub_config.py etc.

```suggestion
            - mountPath: /etc/jupyterhub/z2jh.py
```